server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -1691,7 +1693,7 @@ protected TSStatus executeNonQueryPlan(PhysicalPlan plan) {
     } catch (QueryProcessException e) {
       logger.error("meet error while processing non-query. ", e);
       Throwable cause = e;
-      while (cause.getCause() != null){
+      while (cause.getCause() != null && cause.getCause().getCause() != null) {
```

Comment:
Could you please explain this?

Reply:
this is used to fix the UT tests that the exception only shows the message bug but lacks the exception type. I saw you have fixed in the commit[1], but I think it's better to be consistent with the single server.
So if you agree with me, I would re-add the exception type you've deleted in [1], otherwise, I'll change the code here as normal.
[1] https://github.com/apache/iotdb/commit/83e916a9e7bc618f348b7078bd79e62db6d9d546

Reply:
have changed the code as normal

Reply:
I think it would be better to think from the user side. Such exceptions does few meaning to users, so we'd better remove them for clarity.